File: tcg-java/test/de/kimminich/kata/tcg/syntactic/StrategySugar.java
package de.kimminich.kata.tcg.syntactic;

import de.kimminich.kata.tcg.Card;

import java.util.List;

public class StrategySugar {

    public static int withMana(int mana) {
        return mana;
    }

    public static int andHealth(int health) {
        return health;
    }

    public static List<Card> fromCards(Integer... cards) {
        return Card.list(cards);
    }

}

File: tcg-kotlin/src/test/kotlin/de/kimminich/kata/tcg/syntactic/StrategySugar.kt
package de.kimminich.kata.tcg.syntactic

import de.kimminich.kata.tcg.Card

object StrategySugar {
    fun withMana(mana: Int): Int {
        return mana
    }

    fun andHealth(health: Int): Int {
        return health
    }

    fun fromCards(vararg values: Int): MutableList<Card> {
        return MutableList(values.size) { Card(values[it]) }
    }
}
